src/components/error_tooltip/error_tooltip.ts

```diff
@@ -14,6 +15,10 @@ css/* scss */ `
     border-left: 3px solid red;
     padding: 10px;
   }
+
+  .o-popover:has(.o-error-tooltip) {
+    z-index: ${ComponentsImportance.ErrorToolTip} !important;
```

Comment:
no - you do not override a INTENDED generic  behaviour for a specific case unless you can 100% justify it.

Reply:
> no - you do not override a INTENDED generic behaviour for a specific case unless you can 100% justify it.

Actually the problem is that we cannot change the enum for the popover as you said it is used by (menu,filtermenu..) but the error is wrapped in a popover component so it uses the zindex property of popover so to change it specifically for the case i propose we only target to change the popover when has .o-error-tooltip. any suggestion will be appreciated. Thank you.

Reply:
My point was to say that the issue also exists for menu, filterMenu, etc ... although you only specify it for the error tooltip.
I feel like your comment goes the exact opposite way of mine. I meant that you should address the issue for the other components as well - seeing the bigger picture.

So what are the options?
1. assume that the composer should take precedence over any popover  ? > then change popover zindex
2. Maybe every popover should be able to behave differently (what about several that would be on top of the other ? what if one should always take precedence over the others ?) then we would conclude that setting a generic zindex for popover was a mistake and that every component that uses popover should have its own z-index.

Both approaches are theoretically worth considering although I would strongly suggest to go with 1.

Reply:
> My point was to say that the issue also exists for menu, filterMenu, etc ... although you only specify it for the error tooltip. I feel like your comment goes the exact opposite way of mine. I meant that you should address the issue for the other components as well - seeing the bigger picture.
> 
> So what are the options?
> 
> 1. assume that the composer should take precedence over any popover  ? > then change popover zindex
> 2. Maybe every popover should be able to behave differently (what about several that would be on top of the other ? what if one should always take precedence over the others ?) then we would conclude that setting a generic zindex for popover was a mistake and that every component that uses popover should have its own z-index.
> 
> Both approaches are theoretically worth considering although I would strongly suggest to go with 1.

Hey, I’m working on your suggestion, currently I am trying to figure out how we can achieve the same for all the Popover components. I have got few results, but at the same time there are few functional issues I’m trying to solve. Will let you know as soon as I’m ready with all the points.